ethereum/src/builder.rs

```diff
@@ -110,8 +110,8 @@ impl<DB: Database> EthereumClientBuilder<DB> {
         self
     }
 
-    pub fn fallback(mut self, fallback: &str) -> Self {
-        self.fallback = Some(fallback.to_string());
+    pub fn fallback(mut self, fallback: Url) -> Self {
```

Comment:
We should use the Into<Url> pattern that we do in some of the other builders. All builders should take Into<Url> rather than just a Url.